Library/ViewModels/RewardAddOnSelectionViewModelTests.swift

```diff
@@ -467,6 +484,7 @@ final class RewardAddOnSelectionViewModelTests: TestCase {
       |> Reward.lens.remaining .~ nil
       |> Reward.lens.startsAt .~ (MockDate().timeIntervalSince1970 + 30)
       |> Reward.lens.endsAt .~ (MockDate().timeIntervalSince1970 + 60)
+      |> Reward.lens.isAvailable .~ false
```

Comment:
This test actually includes some unavailable rewards, so updating it to reflect that.